src/__tests__/conversion.spec.ts

```diff
@@ -38,6 +41,33 @@ describe('MIDI to frequency converter', () => {
   });
 });
 
+describe('Frequency to MTS converter', () => {
+  it('converts a known value', () => {
```

Comment:
Should be `it('converts known values', ...)`. This test case is doing multiple things.

Reply:
Thanks, I didn't connect the dots on that one.

Reply:
No problem. For the smoothest experience it would be nice if this `it` case was broken into multiple invididual test cases. Now if the first `expect` fails the rest won't run and you have to run the unit tests multiple times when things eventually break.